src/ddscxx/include/org/eclipse/cyclonedds/topic/qos/TopicQosDelegate.hpp

```diff
@@ -248,6 +250,18 @@ TopicQosDelegate::policy<dds::core::policy::Ownership>()
     return ownership_;
 }
 
+
```

Comment:
Please also encapsulate this whole body with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro. (BTW: I see that in this particular case the embedding OMG_DDS_OWNERSHIP_SUPPORT macro was also missing for the "ownership" policy).

Reply:
Will do.